cpp/src/arrow/acero/exec_plan.cc

```diff
@@ -1048,7 +1048,7 @@ Result<std::unique_ptr<RecordBatchReader>> DeclarationToReader(
   return DeclarationToReader(std::move(declaration), std::move(options));
 }
 
-namespace internal {
+namespace acerointernal {
```

Comment:
Similar point here: probably can just be `internal`.

Reply:
reverted